core/test/financial-templates/Liquidatable.js

```diff
@@ -176,7 +176,6 @@ contract("Liquidatable", function(accounts) {
       );
     });
   });
-
```

Comment:
Why did you change the formatting here?

Reply:
we weren't consistently adding newlines between `describe()` test sections. I elected to remove all newlines. I think it'd be better on second thought to enforce newlines after each one.